docs/canary-config.md

```diff
@@ -0,0 +1,100 @@
+    - `remove` - Use when you expect a metric to always have data and you want the NaNs removed from your data set (usage metrics).
+    - `replace` - Use when you expect a metric to return no data in certain use cases and you want the NaNs replaced with zeros (for example: count metrics, if no errors happened, then metric will return no data for that time interval).
+- `critical` **true** (boolean, optional) - Use to fail the entire canary if this metric fails (recommended for important metrics that signal service outages or severe problems).
+- `mustHaveData` **true** (boolean, optional) - Use to fail a metric if data is missing.
+- `effectSize` ([EffectSize](#effect-size), optional) - Controls how much different the metric needs to be to fail or fail critically.
+
+<a name="effect-size"></a>
+### EffectSize
+Controls the degree of statistical significance the metric needs to fail or fail critically. 
+Metrics marked as critical can also define `criticalIncrease` and `criticalDecrease`. 
+See the [Netflix Automated Canary Analysis Judge] and [Mann Whitney Classifier] classes for more information.
+
+#### Properties
+- `allowedIncrease` **1.1** (number, optional) - Defaults to 1. The multiplier increase that must be met for the metric to fail. This example makes the metric fail when the metric has increased 10% from the baseline.
+- `allowedDecrease` **0.90** (number, optional) - Defaults to 1. The multiplier decrease that must be met for the metric to fail. This example makes the metric fail when the metric has decreased 10% from the baseline.
+- `criticalIncrease` **5.0** (number, optional) - Defaults to 1. The multiplier increase that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 5x increase.
+- `criticalDecrease` **0.5** (number, optional) - Defaults to 1. The multiplier decrease that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 50% decrease.
+
+<a name="canary-classifier-config"></a>
+### CanaryClassifierConfig
+#### Properties
+- `groupWeights` (enum[string], required)
+  - `groups` **"Latency" : 33** (object, required) - List of each metrics group along with its corresponding weight. Weights must total 100.
+  
+<a name="links"></a>
+## Links
+- [Spinnaker Canary Best Practices]
+- [Canary analysis: Lessons learned and best practices from Google and Waze]
+- [Armory Kayenta Documentation]
+- [Example Signalfx canary config]
+  
+[Spinnaker Canary Best Practices]: https://www.spinnaker.io/guides/user/canary/best-practices/
+[Armory Kayenta Documentation]: https://docs.armory.io/spinnaker/configure_kayenta/
+[Example Signalfx canary config]: https//github.com/spinnaker/kayenta/blob/master/kayenta-signalfx/metric-query-config.md
```

Comment:
```suggestion
[Example Signalfx canary config]: https://github.com/spinnaker/kayenta/blob/master/kayenta-signalfx/metric-query-config.md
```

Reply:
Sorry, I should have checked my own suggestion to be typo free!

Reply:
🥇